jsonschema/validators.py

```diff
@@ -641,28 +673,28 @@ def resolving(self, ref):
 
         """
 
-        url, resolved = self.resolve(ref)
-        self.push_scope(url)
+        uri, resolved = self.resolve(ref)
+        self.push_scope(uri)
         try:
             yield resolved
         finally:
             self.pop_scope()
 
     def resolve(self, ref):
-        url = self._urljoin_cache(self.resolution_scope, ref)
-        return url, self._remote_cache(url)
+        uri = self._urijoin_cache(self.resolution_scope, ref)
```

Comment:
See above, this seems like just diff churn, and can't anyhow be renamed for backwards compatibility reasons.

Reply:
(y)